deploy/examples/operator-openshift.yaml

```diff
@@ -815,13 +815,13 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
 
-          # Recommended resource requests and limits, if desired
-          #resources:
-          #  limits:
-          #    memory: 512Mi
-          #  requests:
-          #    cpu: 100m
-          #    memory: 128Mi
+          resources:
+            limits:
+              cpu: 1
```

Comment:
Can we avoid a cpu limit? Since cpu is a resource that is lost if not used in the moment, burstable components like operators will benefit from not having a limit, and be allowed to use all available unused cpu. This does not affect other components, since all components are guaranteed their cpu requests, this is just about using spare cpu. 
```suggestion
```